### PR TITLE
[ZH] Translate 13-1.md

### PR DESCRIPTION
**Pull request by sueqian6**

### CODE REVIEW

File: docs/zh/week13/13_1.md [hunk 1]
```diff
@@ -0,0 +1,377 @@
+---
+lang-ref: ch.13-1
+title: Graph Convolutional Networks I
```
Comment (Atcold) [inline docs/zh/week13/13_1.md:3]: `lang: zh` is missing.

**Review by Atcold — CHANGES_REQUESTED** [review]
The file name should be `13-1.md` not `13_1.md`. Please, `git mv 13_1.md 13-1.md` and `git push`.
Also, you need to update `docs/_config.yml` accordingly.

File: docs/zh/week13/13_1.md (hunk 1 above)
Comment (sueqian6) [reply docs/zh/week13/13_1.md]: > `lang: zh` is missing.

It seems I can't edit the file now. The 'edit' button directs me to 404.

**Review by Atcold — CHANGES_REQUESTED** [review]
Sweet. Just edit `docs/_config.yml` in this branch, and then we're good to go.

File: docs/zh/week13/13_1.md (hunk 1 above)
Comment (Atcold) [reply docs/zh/week13/13_1.md]: Now it's correct. I think you've managed!

**Pull request merged**